spec/features/user_dashboard_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'spec_helper'
+
+feature 'When I am in the dashboard' do
+
+  context 'as a logged in user' do
+    let(:user) { create(:user) }
+    let!(:service_requests) { create_list(:service_request, 3) }
+
+    background do
+      sign_in_user user
+    end
+
+
```

Comment:
@abisosa  porque se quitaron todas las votaciones 
scenario 'I can vote on an service_request', js: true do
      visit root_path
      within first(".js-vote_status") do
        click_link "VOTA"
      end
      expect(page).to have_content('Votaste')
    end
Por ahora esa prueba no se ocupa